package.json

```diff
@@ -28,18 +28,23 @@
   "bugs": "http://github.com/ncb000gt/node-es/issues",
   "devDependencies": {
     "chai": "*",
-    "coveralls": "*",
-    "jscoverage": "0.3.8",
-    "jshint": "*",
+    "eslint": "~1.10.3",
+    "istanbul": "~0.4.2",
+    "mkdirp": "*",
     "mocha": "*",
-    "mocha-lcov-reporter": "*",
+    "rimraf": "*",
     "stact": "*"
   },
   "scripts": {
-    "coverage": "rm -rf ./reports ; mkdir -p ./reports ; NODE_ELASTICSEARCH_COVERAGE=true mocha -R html-cov -r ./test/common.js -u bdd ./test/lib > reports/coverage.html",
-    "functional": "mocha --check-leaks -R spec -r ./test/common.js -u bdd ./test/functional",
-    "pretest": "jshint *.js ./lib/*.js ./test/*.js ; jscoverage ./lib ./lib-cov",
-    "test": "mocha --check-leaks -R spec -r ./test/common.js -u bdd ./test/lib ./test/functional",
-    "posttest": "NODE_ELASTICSEARCH_COVERAGE=true mocha -R mocha-lcov-reporter -r ./test/common.js -u bdd ./test/lib | ./node_modules/coveralls/bin/coveralls.js"
+    "clean": "rimraf reports && mkdirp reports",
+    "functional": "mocha test/functional",
+    "lint": "eslint index.js lib test",
+    "pretest": "npm run clean",
+    "test": "istanbul cover --root lib --dir reports node_modules/mocha/bin/_mocha -- test/lib test/functional",
+    "posttest": "npm run lint"
+  },
+  "dependencies": {
+    "bluebird": "~3.1.1",
```

Comment:
It is, yeah. The graph is thenify-all > thenify > native-or-bluebird -- which doesn't explicitly depend on bluebird (otherwise it would be always included). There's no good way, AFAIK, in Node to conditionally depend on things (which is probably a good thing?)